BondarenkoBE/lab_5/ConsoleApplication20.cpp

```diff
@@ -0,0 +1,139 @@
+// ConsoleApplication20.cpp: ���������� ����� ����� ��� ����������� ����������.
+//
+
+#include "stdafx.h"
+#include <iostream>
+#include <iomanip>  
+#include <conio.h>
+
+using namespace std;
+const int N = 10;
+int m1, m2, n1, n2, m3, n3;
```

Comment:
Do not use global variables. Only const are allowed

Reply:
It's because you make them to work that way.
Knowing that, a can assume that you are able to change it.